samples/word/04-range/split-words-of-first-paragraph.yaml

```diff
@@ -4,157 +4,84 @@ description: 'Show how to split a paragraph into word ranges and then traverses
-                                    return createTimerPromise(200).then(context.sync);
-                                })
-                            })
-                    })
-            })
-                .catch(OfficeHelpers.Utilities.log);
-
-            function createTimerPromise(ms) {
-                return new OfficeExtension.Promise(function (resolve) {
-                    setTimeout(resolve, ms);
-                })
-            }
+    content: |
+        $("#setup").click(() => tryCatch(setup));
+        $("#highlight").click(() => tryCatch(highlightWords));
+
+        async function highlightWords() {
+            await Word.run(async (context) => {
+                let paragraph = context.document.body.paragraphs.getFirst();
+                let words = paragraph.split(
+                    [" "], true /* trimDelimiters*/, true /* trimSpaces */);
+                words.load("text");
+                await context.sync();
+
+                for (let i = 0; i < words.items.length; i++) {
+                    if (i >= 1) {
+                        words.items[i - 1].font.highlightColor = "#FFFFFF";
+                    }
+                    words.items[i].font.highlightColor = "#FFFF00";
+                    await context.sync();
+                    await pause(200);
+                }
+            });
+        }
 
-            function forEach(collection, handler) {
-                var promise = new OfficeExtension.Promise(function (resolve) { resolve(); });
-                collection.items.forEach(function (item, index) {
-                    promise = promise.then(function () {
-                        return handler(item, index);
-                    })
-                });
-                return promise;
-            }
+        function pause(milliseconds) {
+            return new Promise(resolve => setTimeout(resolve, milliseconds));
         }
 
-        function setup() {
-            Word.run(function (context) {
+        async function setup() {
+            await Word.run(async (context) => {
                 // lets insert a couple of paragraphs to illustrate the point..
                 context.document.body.clear();
                 context.document.body.insertParagraph("Video provides a powerful way to help you prove your point. When you click Online Video, you can paste in the embed code for the video you want to add. You can also type a keyword to search online for the video that best fits your document.", "start");
                 context.document.body.paragraphs.getLast().insertText("To make your document look professionally produced, Word provides header, footer, cover page, and text box designs that complement each other. For example, you can add a matching Online cover page, header, and sidebar. Click Insert and then choose the Online elements you want from the different Online galleries.", "replace");
-                return context.sync();
+                await context.sync();
```

Comment:
Same comment as before -- perhaps follow convention of including blank line before and after **await context.sync();**?